### ISSUE TITLE
small doubt

### ISSUE THREAD

**Issue by jacfuu1** [GoogleChrome/lighthouse#6916]
if i write git log --author=paul i get only paul Irish  but i will not get Paul Lewis eg
 $ git log --author=Paul
commit bebcd6f7de13f148fa8d69ca10f2ccfe0a4d402c
Author: Paul Irish <paul.irish@gmail.com>
Date:   Tue Dec 11 15:38:09 2018 -0800

    core(lantern): drop node id from error message (#6774)

commit 9f58eb418bc5bd6c177bfd2d50e16840c23ac16e (tag: v4.0.0-beta)
Author: Paul Irish <paul.irish@gmail.com>
Date:   Thu Nov 29 14:31:40 2018 -0800

#### Provide the steps to reproduce
1. Run LH on <affected url>

#### Environment Information
* Affected Channels: 
* Lighthouse version:
* Node.js version:
* Operating System:

**Related issues**

**Comment by Hoten** [comment]
This bug tracker is for sharing problems using Lighthouse or ideas for improvement. This is not an appropriate use of this channel.

fyi you do get all Paul's. try `git log --author "Paul" | grep -C6 lewis`

**Comment by paulirish** [comment]
The issue tracker here is for bug reports for the Lighthouse project. 
Using the git cli is not in scope for us.. and I'd recommend reading around https://git-scm.com/docs/git-log


All that said.. here you go:

```sh
git log --author=Paul --oneline --format="%an %h %s"  --before="2016-06-15"

Paul Irish de989284 Merge pull request #443 from GoogleChrome/fix442
Paul Lewis e6fa727f Moves online-only html
Paul Lewis cf95bbb3 Updates tests and fixes bugs found by doing so (#441)
Paul Lewis 9daa5017 Screenshot filmstrip (#440)
Paul Lewis 96c81aa4 Fixes prepareAssets (#439)
Paul Irish d19b4044 Merge pull request #435 from zhaoz/master
Paul Lewis c0fecbd9 Audit filtering (#432)
Paul Irish bdf13c65 Merge pull request #429 from GoogleChrome/audit-filter
Paul Lewis c3110db6 Restores filtering audits
....
```

😉
